commands/peers/deletepeer.go

```diff
@@ -3,29 +3,63 @@ package peercommands
 import (
 	"net/http"
 
+	"github.com/gluster/glusterd2/context"
 	"github.com/gluster/glusterd2/peer"
 	"github.com/gluster/glusterd2/rest"
+	"github.com/gluster/glusterd2/rpc/client"
+
+	log "github.com/Sirupsen/logrus"
+	etcdclient "github.com/coreos/etcd/client"
+	etcdcontext "golang.org/x/net/context"
 
 	"github.com/gorilla/mux"
 )
 
 func deletePeerHandler(w http.ResponseWriter, r *http.Request) {
-	p := mux.Vars(r)
+	peerReq := mux.Vars(r)
```

Comment:
i mean to say what this function is doing. we should write comment just above function